### PR TITLE
editor

### PR DESCRIPTION
**Pull request by AlmogShaul**
Form preview page

### CODE REVIEW

File: src/app/editor/editor.component.ts [hunk 1]
```diff
@@ -8,22 +8,20 @@ import { Component } from '@angular/core';
...
 
-  constructor() {
-
-  }
-
-
   fileSelected(data){
     let file : any= data.currentTarget.files[0];
       var reader = new FileReader();
       reader.addEventListener('load', (result) => {
```
Comment (schmuli) [inline src/app/editor/editor.component.ts]: This type of functionality should be moved into a file utils service

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <script src="shim_for_IE.js"></script>
```
Comment (schmuli) [inline src/index.html]: We need to look at what exactly this provides, and if core-js doesn't have it already, but this is okay for now.

File: src/app/editor/editor.component.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import {ChangeDetectorRef, Component} from '@angular/core';
```
Comment (schmuli) [inline src/app/editor/editor.component.ts]: Are you using the `ChangeDetectorRef` here?

File: src/app/editor/editor.component.ts (hunk 1 above)
Comment (AlmogShaul) [inline src/app/editor/editor.component.ts]: The parsing?

File: src/app/editor/editor.component.ts (hunk 3 above)
Comment (AlmogShaul) [inline src/app/editor/editor.component.ts]: will b removed

File: src/app/editor/editor.component.ts (hunk 1 above)
Comment (AlmogShaul) [inline src/app/editor/editor.component.ts]: If so,
I think it should be a static utility file

Comment (AlmogShaul) [inline src/app/editor/editor.component.ts]: @schmuli ^

File: src/app/eform/preview/preview.component.less [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  right: 0;
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+  cursor: pointer;
+  opacity: 0;
+  filter: alpha(opacity=0);
+}
+
+.sf-form {
```
Comment (schmuli) [inline src/app/eform/preview/preview.component.less]: You should need `sf-form` here, did you mean `ef-form`?

**Pull request merged**